src/gui/mzroll/main.cpp

```diff
@@ -71,7 +71,7 @@ int main(int argc, char *argv[])
     qApp->setApplicationVersion(STR(EL_MAVEN_VERSION));
 
 
-    elmavexceptionhandler::init();
```

Comment:
@saifulbkhan Why is this commented out?

Reply:
Sorry about this. The change should not have been staged. Fixing.